source/includes/api/_query.md

```diff
@@ -162,43 +388,3 @@ You can delimit an area of interest by providing a geostore id as a parameter:
 ```shell
 curl -i -XGET http\://api.resourcewatch.org/v1/query/b99c5f5e-00c6-452e-877c-ced2b9f0b393\?sql\=SELECT\ \*\ from\ nexgddp-historical-ACCESS1_0-prmaxday\ \ where\ year\ between\ 1960\ and\ 1962&geostore\=0279093c278a64f4c3348ff63e4cfce0
```

Comment:
there are a few example curls here that are explicitly escaped, and seem to be the only ones on the docs. could you un-escape them please? thanks

Reply:
Sure!